lib/emailer.js

```diff
@@ -4,16 +4,18 @@ const path = require('path');
 const nodemailer = require('nodemailer');
 const stubTransport = require('nodemailer-stub-transport');
 const smtpTransport = require('nodemailer-smtp-transport');
+let transport;
```

Comment:
On second thoughts this might work better if the nodemailer-ses-transport was an option that one could switch to. So I might trigger nodemailer-ses-transport by passing the option 'ses'